backend/spec/system/front_end/question_views_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'Question', type: :system do
+  let!(:user) { create(:user_student) }
+  let!(:question) { create(:multiple_choice_question) }
+
+  describe '#show' do
+    before do
+      sign_in user
+      visit question_path question
```

Comment:
I think that is not part of this PR but should we display only questions in some cases, please create an issue for this

Reply:
It should show the first question, right?